doc/user_guide/eels.rst

```diff
@@ -205,7 +205,7 @@ core-loss file
 
 .. code-block:: python
 
-    >>> s.set_microscope_parameters(beam_energy=100,
+    >>> s.set_microscope_parameters(beam_energy=300,
```

Comment:
Actually, it seems that these information are already in the metadata! Remove it to simplify the user guide, since this is explained elsewhere?

Reply:
I think `set_microscope_parameters` is in the user guide to show the function exits. I was just confused, that my quantification results were way off when running the whole user guide example for BN. As you suggested, we can remove it here as all all parameters are set in the EELSdb BN spectrum and just mention `set_microscope_parameters` in the text.